src/Components/test/testassets/ComponentsApp.App/Shared/NavMenu.cshtml

```diff
@@ -22,6 +22,11 @@
                 <span class="oi oi-list-rich" aria-hidden="true"></span> Fetch data
```

Comment:
Didn't you delete this component? Should the link also be removed? Or maybe it would be better to undelete the `FetchData` component and retain both.

Reply:
Ahh, that was accidental. Undone